ground/src/main/res/drawable/task_chip_image_button.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2023 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <size
+    android:width="20dp"
+    android:height="20dp" />
+  <solid android:color="#FFFFFF" />
```

Comment:
Should this be a reference to a color resource?

Reply:
This color is for background. I don't feel too strongly about moving this to colors.xml as it is independent of which button theme is being used (dark green, light green or outlined)

Reply:
Well I think it's probably worth having a colorBackground resource in case we end up changing the background color at some point. Maybe we will eventually want it to be very faintly tinted with the primary color and then we would only have to update the one resource to do that.

Reply:
Done